MerlinAU.sh

```diff
@@ -1286,11 +1287,19 @@ _CreateEMailContent_()
            ;;
        STOP_FW_UPDATE_APPROVAL)
            emailBodyTitle="WARNING"
+           high_risk_terms="factory default reset|features are disabled|break backward compatibility|must be manually|strongly recommended"
```

Comment:
This is to satisfy the request from [nlurker](https://www.snbforums.com/members/nlurker.60367/)
Here: https://www.snbforums.com/threads/merlinau-v1-1-3-the-ultimate-firmware-auto-updater-now-available-in-amtm.88577/post-908680

In this code, if it's set as HTML it highlights it yellow:

![image](https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/assets/1971404/70f3ccfc-5d39-4b6f-ad3c-bdf20bb4b3ca)

Else if it's plain text it tags it with asterisks:

![image](https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/assets/1971404/67a6c06d-908a-4554-b2cc-6fce71223a8b)

Reply:
> This is to satisfy the request from [nlurker](https://www.snbforums.com/members/nlurker.60367/) Here: https://www.snbforums.com/threads/merlinau-v1-1-3-the-ultimate-firmware-auto-updater-now-available-in-amtm.88577/post-908680
> 
> In this code, if it's set as HTML it highlights it yellow:
> ...
> Else if it's plain text it tags it with asterisks:
> 
Nice!!